core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -3898,6 +3900,9 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
             CHECK_SUSPEND_FLAGS();
 #endif
         }
+#if WASM_ENABLE_THREAD_MGR != 0
+        CHECK_SUSPEND_FLAGS();
```

Comment:
Had better remove L3899 to L3901 to avoid duplicated check after calling non-import function.